.babelrc

```diff
@@ -1,11 +1,8 @@
 {
-    "sourceMaps": "inline",
+    "sourceMaps": true,
```

Comment:
with `true` we just let babel decide what is best for us. When we need inline sourcemaps we ask for them.